physics/drag_suite.F90

```diff
@@ -395,29 +397,40 @@ subroutine drag_suite_run(                                           &
 ! Each component is tapered off automatically as a function of dx, so best to
 ! keep them activated (.true.).
       logical, intent(in) ::   &
-      do_gsl_drag_ls_bl,       & ! large-scale gravity wave drag and blocking
+      do_gsl_drag_ls_bl,       & ! mesoscale gravity wave drag and blocking
```

Comment:
I kept the "_ls_" designation in the "_do_gsl_drag_ls_bl_" variable because this is a namelist variable that is already hardwired in multiple workflows that use the Unified UGWP suite.  Just avoiding the need to have these workflows being updated.